plugins/queue/scheduler.js

```diff
@@ -221,7 +247,7 @@ async function startPeriodic(executor, config) {
     }
     logger.info(`added to delayed queue for job ${job.id}`);
 
-    if (triggerBuild && !job.archived) {
+    if (triggerBuild && (periodicBuildTableEnabled || !job.archived)) {
```

Comment:
```suggestion
    if (triggerBuild && !job.archived) {
```
what it was is fine for checking archived job and is not really related to `periodicBuildTable`